extensions/amp-lightbox/0.1/amp-lightbox.js

```diff
@@ -15,11 +15,13 @@
  */
 
 import * as st from '../../../src/style';
+import {ActionTrust} from '../../../src/action-trust';
 import {AmpEvents} from '../../../src/amp-events';
 import {CSS} from '../../../build/amp-lightbox-0.1.css';
 import {Gestures} from '../../../src/gesture';
 import {KeyCodes} from '../../../src/utils/key-codes';
 import {Layout} from '../../../src/layout';
+import {createCustomEvent} from '../../../src/event-helper';
```

Comment:
please move up for alphabetical sort (the reason the lint is failing on the CI precheck)

Reply:
Thank You

Reply:
You can use `gulp lint --fix` for that too. =)

Reply:
@cathyxz thank you for that